### PR TITLE
Black_formatting

### PR DESCRIPTION
**Pull request by CaibinSh**
Codes formatted with black formatter

### CODE REVIEW

**Comment by fgypas** [comment]
@CaibinSh You could try to add the following: `setuptools<=59.5.0`

**Comment by CaibinSh** [comment]
> 



> @CaibinSh You could try to add the following: `setuptools<=59.5.0`

@fgypas Thanks very much! This drives me crazy... 
Btw, where to add it? In the env.yaml?

**Comment by fgypas** [comment]
> > 
> 
> > @CaibinSh You could try to add the following: `setuptools<=59.5.0`
> 
> @fgypas Thanks very much! This drives me crazy... Btw, where to add it? In the env.yaml?

Try in the `scar-cpu.yml` first.

**Comment by fgypas** [comment]
Still the same. Let's ask on gitter...

**Comment by CaibinSh** [comment]
> 
Unfortunately, the error stays. It seems that it gets stuck here:

CondaHTTPError: HTTP 404 NOT FOUND for url <https://conda.anaconda.org/pytorch/noarch/pytorch-mutex-1.0-cpu.tar.bz2>
Elapsed: 00:00.062381
CF-RAY: [702](https://github.com/Novartis/scAR/runs/6182097937?check_suite_focus=true#step:5:702)1b2148cea5ca7-IAD

An HTTP error occurred when trying to retrieve this URL.
HTTP errors are often intermittent, and a simple retry will get you on your way.


fonttools-4.33.2     | 1.6 MB    | ########## | 100% 
Error: Process completed with exit code 1.

But, let us solve it tmw. I do not want to ruin your wonderful night... 

Cheers, CS

**Comment by fgypas** [comment]
@CaibinSh Others have issues in general these days.

**Comment by CaibinSh** [comment]
> 

Hi @fgypas , it works now. Please feel free to approve this PR. There are mainly code refactoring, I run black and pylint in VSCode and refactor the codes based on the warning info in VScode (Thank you @Tobias-Ternent, I start using VScode now, so wonderful tool!!! It is faster than Atom).

File: scar-cpu.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ channels:
 - pytorch
 - conda-forge
 dependencies:
+- setuptools<=59.5.0
```
Comment (fgypas) [inline scar-cpu.yml:7]: Here try to remove it again and see if it is still working...
If this is required then add it in the scar-gpu.yml as well

**Review by fgypas — APPROVED** [review]
Just make sure if you need the specific version of setuptools or not. I left a comment on this.

**Comment by CaibinSh** [comment]
> Just make sure if you need the specific version of setuptools or not. I left a comment on this.

We probably do not need to specify it. I have removed it from scar-cpu.yml

**Pull request merged**